webapp/channels/src/components/file_upload_overlay.tsx

```diff
@@ -32,9 +31,9 @@ const FileUploadOverlay: React.FC<Props> = (props: Props) => {
                         alt='Files'
                     />
                     <span>
-                        <LocalizedIcon
+                        <i
                             className='fa fa-upload'
-                            title={{id: t('generic_icons.upload'), defaultMessage: 'Upload Icon'}}
+                            title={formatMessage({id: 'generic_icons.upload', defaultMessage: 'Upload Icon'})}
```

Comment:
As this component didnt specify which tag to use, the default tag should be `i` tag.

Notice the removal of `t` function and addition of `formatMessage`